app/assets/stylesheets/app/components/lightbox.scss

```diff
@@ -36,47 +36,23 @@ body.lightbox-open,
 	}
 
 	.lightbox-dialog {
-	width: auto;
-	padding: 10px;
-	margin-right: auto;
-	margin-left: auto;
-
-	.lightbox-header {
-		float: right;
-		.close {
-			margin-top: -2px;
-		}
-	}
-
-	.lightbox-content {
-		position: relative;
-		background-color: #ffffff;
-		border: 1px solid #999999;
-		border: 1px solid rgba(0, 0, 0, 0.2);
-		border-radius: 6px;
-		outline: none;
-		-webkit-box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
-		box-shadow: 0 3px 9px rgba(0, 0, 0, 0.5);
-		background-clip: padding-box;
-		/**/
+		width: auto;
 		padding: 10px;
-		display: inline-block;
-
-		.lightbox-caption {
-			position: absolute;
-			right: 8px;
-			bottom: 8px;
-			left: 10px;
-			padding: 2%;
-			font-size: 14px;
-			line-height: 18px;
-			color: white;
-			text-align: center;
-			text-shadow: 0 -1px 0 #000;
-			text-shadow: 0 -1px 0 rgba(0,0,0,0.3);
-			background: #000;
-			background: rgba(0,0,0,0.6);
+		margin-right: auto;
+		margin-left: auto;
+
+		.lightbox-content {
```

Comment:
Rule set contains (11/10) properties<br>Selector should have depth of applicability no greater than 2, but was 3